test/xgo_integration/check.go

```diff
@@ -0,0 +1,35 @@
+package xgo_integration
+
+import (
+	"os"
+	"os/exec"
+	"sync"
+	"testing"
+)
+
+// must have xgo installed first
+// otherwise skipped
+
+var checkXgoOnce sync.Once
+var checkXgoErr error
```

Comment:
This one could have been a Boolean, you set in the Once

Then you test the boolean


```suggestion
var xgoFound bool
```

I wonder why you added in packages scoped variable until I noticed you only wanted to check if you had an error

This would only be OK if you agree the lookup error has no interest.

Also you could report it once, then skip without mentioning the error

Reply:
I also want to `t.Skipf(err)`